themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```diff
@@ -515,8 +515,12 @@ a.zocial  {
     width: 100%;
 }
 
-.login-pf-page .card-pf{
-  margin-bottom: 10px;
+.otp-tile {
+    padding: 1.5rem;
```

Comment:
Is it possible to add any `margin-bottom`? If user have defined f.e. 3 OTP credentials with longer name, there are two rows, but there are no space between them.

Reply:
@mabartos (cc: @vmuzikar ) fixed for IE (see the screenshot above) and other browsers:
![obrázok](https://user-images.githubusercontent.com/12138171/97352769-5e858580-1893-11eb-98e4-747aa9c87850.png)